CarCostsTracker/CarCostsTracker/History/HistoryCollectionViewCell.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  HistoryCollectionViewCell.swift
+//  CarCostsTracker
+//
+//  Created by pavels.vetlugins on 02/10/2018.
+//  Copyright © 2018 Ilyas-Karshigabekov. All rights reserved.
+//
+
+import UIKit
+
+class HistoryCollectionViewCell: UICollectionViewCell {
+    @IBOutlet weak var costsLogo: UIImageView!
+    @IBOutlet weak var costsType: UILabel!
+    @IBOutlet weak var milage: UILabel!
+    @IBOutlet weak var date: UILabel!
+    @IBOutlet weak var detailedDescription: UILabel!
```

Comment:
Rename every outlet to have type of object in the end of the name, example:
`costLogo` -> `costLogoImageView`.
Typo in `milage` should be `mileage`.

Reply:
Done